apps/studio/src/pages/home/wallet/Wallet.tsx

```diff
@@ -89,76 +99,120 @@ const Wallet: FunctionComponent = () => {
     setTab(nextTab);
   };
 
-  if (!wallet) {
-    return <NoConnectedWallet />;
-  }
-
-  return (
-    <Container maxWidth={ false }>
-      <Box mx={ [null, null, 1.5] }>
-        <Box
-          sx={ {
-            alignItems: "center",
-            display: "flex",
-            justifyContent: "space-between",
-            mb: 5,
-          } }
-        >
-          <Typography fontWeight={ 800 } variant="h3">
-            WALLET
-          </Typography>
-          <DisconnectWalletButton />
+  if (isClientConfigLoading) return;
+
+  if (!isClaimWalletRoyaltiesEnabled) {
+    return (
+      <Container maxWidth={ false }>
+        <Box mx={ [null, null, 3] }>
+          <Box
+            sx={ {
+              alignItems: "center",
+              display: "flex",
+              justifyContent: "space-between",
+              mb: 5,
+            } }
+          >
+            <Typography fontWeight={ 800 } variant="h3">
+              WALLET
+            </Typography>
+            { wallet ? (
+              <DisconnectWalletButton />
+            ) : (
+              <Button
+                gradient="crypto"
+                sx={ { mb: 5, mr: [0, 4.75] } }
+                width="compact"
+                onClick={ () => dispatch(setIsConnectWalletModalOpen(true)) }
+              >
+                Connect Wallet
+              </Button>
+            ) }
+          </Box>
+          <Box mt={ 5 } pb={ 5 }>
+            <Typography variant="body1">
+              Wallet Portfolio and Transactions Coming Soon
+            </Typography>
+          </Box>
         </Box>
```

Comment:
Added this section to handle when the feature flag is false so it still covers the connect wallet purpose for the Wallet page in Studio.